packages/amplify-e2e-tests/src/__tests__/export-pull-b.test.ts

```diff
@@ -5,7 +5,7 @@ import {
   addApiWithoutSchema,
   addAuthWithMaxOptions,
   addConvert,
-  addDEVHosting,
+  addPRODHosting,
```

Comment:
This looks unused, did you mean to convert the function below to use PROD or is this import unnecessary?

Reply:
This import is unnecessary. I did not want to addProdHosting because that takes a lot of time to spin up and tear down considering that test is not testing hosting specifically.